components/identities/actions/identityActions.tsx

```
@@ -29,345 +29,394 @@ import { StarknetIdJsContext } from "@/context/StarknetIdJsProvider";
-          hasReverseAddressRecord,
-          callDataEncodedDomain
-        )
-      : [],
-  });
+	const { sendAsync: setMainId, data: mainDomainData } = useSendTransaction({
+		calls: identity
+			? identityChangeCalls.setAsMainId(
+					identity,
+					hasReverseAddressRecord,
+					callDataEncodedDomain
+			  )
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)